tools/minify-js

```diff
@@ -45,11 +45,11 @@ def get_changed_source_files(other_checkout):
     try:
         git_dir = os.path.join(other_checkout, '.git')
         old_commit_sha1 = subprocess.check_output(['git', 'rev-parse', 'HEAD'],
-                                                  env={'GIT_DIR': git_dir})
+                                                  env={'GIT_DIR': git_dir}, universal_newlines=True)
```

Comment:
Well I think of commit IDs as text, since they are ascii-only and meant to be human readable. `hashlib._hashlib.HASH.hexdigest` also returns text, not bytes. If you really think it should be bytes I'll change it.